k9mail-library/src/main/java/com/fsck/k9/mail/internet/MessageExtractor.java

```diff
@@ -447,24 +447,14 @@ private static Boolean isPartTextualBody(Part part) throws MessagingException {
             dispositionFilename = MimeUtility.getHeaderParameter(disposition, "filename");
         }
 
-        boolean isAttachmentDisposition = "attachment".equalsIgnoreCase(dispositionType) || dispositionFilename != null;
```

Comment:
For this one I added a comment instead of creating a variable; is that acceptable or would you prefer the variable?

Reply:
We generally try avoid comments. It's easy to change code without updating comments. Giving a thing a descriptive name is often the easiest way to avoid a comment.